sklearn/externals/joblib/format_stack.py

```diff
@@ -264,7 +263,11 @@ def tokeneater(token_type, token, start, end, line):
         # dotted names
         tokeneater.name_cont = False
 
+            
+           
+
```

Comment:
Please do not introduce unnessary blank lines. Please run the [pep8](https://pypi.python.org/pypi/pep8) check tool when not sure.